NOAA-EMC/global-workflow
# Add efcs and epos to ufs_hybatm xml (#3192)

# Description
g-w PR #3185 inadvertently removed metatasks enkfgdas_fcst and enkfgdas_epmn from the ufs_hybatmDA xml.  This PR modifies `workflow/applications/gfs_cycled.py` so that these metatasks are once again in the ufs_hybatmDA xml.

Resolves: #3192 

# Type of change


# Change characteristics
- Does this change require a documentation update? **NO**
- Does this change require an update to any of the following submodules? **NO**


# How has this been tested?
1. clone and install `bugfix/ufs_hybatmDA` on Dogwood
2. set up g-w CI C96C48_ufs_hybatmDA by executing `$HOMEgfs/workflow/create_experiment.py`
3. open C96C48_ufs_hybatmDA xml
4. confirm that metatasks enkfgdas_fcst and enkfgdas_epmn are defined


# Checklist
